tests/sentry/incidents/endpoints/test_organization_alert_rule_index.py

```diff
@@ -989,6 +989,31 @@ def test_no_perms(self):
         resp = self.get_response(self.organization.slug, **self.alert_rule_dict)
         assert resp.status_code == 403
 
+    def test_team_admin_create(self):
```

Comment:
should there be a test for a team admin trying to create an alert for a project they don't have access to?